chart/Chart.yaml

```diff
@@ -15,11 +15,11 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.5
+version: 0.1.6
```

Comment:
With the complete deprecation of `policy/v1beta1` (and potentially deprecation of HPA as well, discussed above) I feel like this should really get a bump to `0.2.0`?  I guess those deprecations happen inside the operator itself and not the chart, so I get the patch bump.  That being said changing from a patch appVersion to a major version bump seems like it should come with a higher chart bump as well.  just my opinion though feel free to disregard.

Reply:
Makes sense. Ill just get another review on this one from @nitisht .

Reply:
Bumped to 0.2.0